### DISCUSSION TITLE
Unable to call the power flow calculation results of pandapower

### DISCUSSION THREAD

**Discussion by sghqsads** [e2nIEE/pandapower#1858] [Q&A]
Hi everyone,

I'm a beginner of pandapower and I have a problem that I could not call the power flow calculation results of pandapower. 

The power flow calculation is right and I can see the results in Jupyter notebook. However, when I use net.res_gen to call the calculation result, it only shows 	p_mw	q_mvar	va_degree	vm_pu.

I would appreciate if someone could give me some advice. Thanks!

BTW. I use Jupyter notebook and my pyhton version is 3.8 inside anaconda, the pandapower version is 2.7.0.

**Comment by MacsBO** [comment]
Hi @sghqsads , I don't understand your question: after you run a power flow you have several results.
Inside net.res_gen you can only see the generators power, the "standard" results of a PF are in net.res_line (currents) , net.res_bus (voltages) and net.res_trafo (or net.res_trafo3w)

**Reply by sghqsads** [reply]
Thank you! My problem is that although Pandapower successfully solved the OPF, I could not see the results by using net.res_line (currents) , net.res_bus (voltages) and net.res_trafo (or net.res_trafo3w).  For instance, after using  net.res_gen, the results are only p_mw q_mvar va_degree vm_pu in the first line, no other values and numbers. I think maybe this problem is in relationship with the setting of my Anoconda and Jupyter notebook.
